### PR TITLE
feat: Introduce activationThreshold/minMetricValue for Predictkube Scaler

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: Jorge Turrado <jorge_turrado@hotmail.es>

This PR also fixes a panic in case of problems related with wrong scaler initialization

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] Tests have been added
- [x] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)*

Related to https://github.com/kedacore/keda/issues/2800
Related to https://github.com/kedacore/keda-docs/pull/838
Fixes https://github.com/kedacore/keda/issues/3247

### CODE REVIEW

File: tests/helper/prometheus/prometheus_helper.go [hunk 1]
```diff
@@ -0,0 +1,510 @@
+//go:build e2e
+// +build e2e
+
+package helper
```
Comment (zroubalik) [inline tests/helper/prometheus/prometheus_helper.go:4]: I'd put this prometheus related helper next to the prometheus scaler in the same package and not into this generic helper package.

Comment (JorTurFer) [reply tests/helper/prometheus/prometheus_helper.go:4]: the problem is that it's used for predictkube and also for prometheus, that's why I moved to the shared helpers

Comment (JorTurFer) [reply tests/helper/prometheus/prometheus_helper.go:4]: I mean, I cannot put inside predictkube package because I'll need on prometheus package too

Comment (zroubalik) [reply tests/helper/prometheus/prometheus_helper.go:4]: :(

**Review by zroubalik — APPROVED** [review]
Great job!

**Pull request merged**